assets/xml/objects/object_test3.xml

```diff
@@ -1,78 +1,88 @@
 ﻿<Root>
+    <!-- Object for Kafei -->
     <File Name="object_test3" Segment="6">
+        <!-- Textures -->
         <Texture Name="object_test3_TLUT_000000" OutName="tlut_000000" Format="rgba16" Width="16" Height="16" Offset="0x0" />
         <Texture Name="object_test3_TLUT_000200" OutName="tlut_000200" Format="rgba16" Width="16" Height="16" Offset="0x200" />
```

Comment:
These TLUTs are used all over the place. `"object_test3_TLUT_000000` is used for:
- `gKafeiClothesPatternTex`
- `gKafeiSkinTex`
- His boot texture
- `gKafeiClothesNeckOpeningTex`
- `gKafeiNoseTex`
- `gKafeiFingersTex`

`object_test3_TLUT_000200` is used for:
- `gKafeiTrouserHemTex`
- `gKafeiHair1Tex`
- `gKafeiHair2Tex`
- `gKafeiSleevePatternTex`
- `gKafeiInnerSleeveTex`

Given that they have a lot of varied usages, I think something like `gKafeiBody1TLUT` and `gKafeiBody2TLUT` probably works best, unless someone else has a better way of describing them.